src/view/Office.vue

```diff
@@ -337,6 +337,11 @@ export default {
 		flex-direction: column;
 		background-color: var(--color-main-background);
 		transition: opacity .25s;
+
+		.viewer & {
```

Comment:
Is this `&` needed?

Reply:
@vinicius73 It does not works without it :grin: because `#richdocuments-wrapper` is not a parent of `.viewer` in the DOM.

This rule means it will only affect `#richdocuments-wrapper` if found inside `.viewer`.